Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo

```diff
@@ -65,6 +65,18 @@ initial equation
    equivalent air temperature calculation is close to 0.
    If there are no walls, windows and ground at all, this might be
    irrelevant.", level=AssertionLevel.warning);
+
+equation
+  delTEqLW=(TBlaSky-TDryBul)*alphaRad/(alphaRad+alphaWallOut);
+  delTEqSW=HSol*aExt/(alphaRad+alphaWallOut);
+  if withLongwave then
+    TEqWin=TDryBul.+delTEqLWWin*abs(sunblind.-1);
```

Comment:
True, `abs()` is not necessary here. But as `sunblind` is a vector, it ends in:

`TEqWin=TDryBul.+delTEqLWWin*(ones(n)-sunblind);`

So one function call for the other. But it looks more straight ahead.